### ISSUE TITLE
Windows + libsoup 3.x

### ISSUE THREAD

**Issue by hmaarrfk** [conda-forge/libsoup-feedstock#46]
### Comment:

I understand that libsoup 3.x requires nghttp2. I have helped compile that. 

However, today, the blocker is on gir gobject-introspection not working correctly.

Would it be ok to release libsoup3 without gir for windows?

xref: https://github.com/conda-forge/libsoup-feedstock/pull/45

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Comment by hmaarrfk** [comment]
In a previous build the following used to be used to copy a known working of the files over:

```
:: The gir files produced by the build are currently broken, don't know why...
copy %RECIPE_DIR%\gir\*typelib %LIBRARY_LIB%\girepository-1.0\
if errorlevel 1 exit 1
copy %RECIPE_DIR%\gir\*gir %LIBRARY_PREFIX%\share\gir-1.0\
if errorlevel 1 exit 1

```

**Comment by hmaarrfk** [comment]
Perhaps this can be a clue https://github.com/conda-forge/librsvg-feedstock/blob/main/recipe/bld.bat

**Comment by tschoonj** [comment]
I don't think we should release a Windows build of libsoup 3 without gobject-introspection, as it feels like a step backwards from what libsoup 2 offered.

However, I won't be the one putting in the fix due to a lack of time and resources, so if you insist we proceed then I am ok with that.

**Comment by hmaarrfk** [comment]
>  feels like a step backwards from what libsoup 2 offered.

I get it. but honestly, given the time resources we have, it isn't all that meaningful to keep building two branches, especially when one is of quite an old version.

PS. a made a few PRs to your feedstocks to help keep libraries in sync. Let me know if you need my help maintaining by adding me as a maintainer there.

**Comment by hmaarrfk** [comment]
i guess the ship has sailed and windows is built without gobject-introspection
